google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/scrape/metadata_scraper.py

```diff
@@ -86,3 +186,18 @@ def _create_rdbms_connection(self, connection_args):
     @classmethod
     def _get_metadata_from_csv(cls, csv_path):
         return pd.read_csv(csv_path)
+
+    def _get_query_assembler(self):
```

Comment:
`_get_query_assembler` and `_execute_refresh_query` are required if you are using the query user config.

Since the rdbms is a commons library, I think this should be in the rdbms README.md for guidance on implementing new RDBMS connectors. WDYT?

Reply:
Sure, I'll add this information

Reply:
I looked at it now, and I'm thinking it actually makes sense to add some context about configurable queries in general into the README.